practice/src/css/components/card.css

```diff
@@ -67,7 +67,7 @@
   object-fit: cover;
 }
 
-.card-product:nth-child(n + 5) {
+.card-product:nth-child(n + 4) {
```

Comment:
I am not sure why we should implement by this way?

Reply:
Because I want to have the `card-product` elements starting from the fourth element onwards not be displayed on the web page. But if you have a better way, can you suggest it to me or I will research further

Reply:
pls go ahead, I will discuss this with you and give suggestions if any